marketplace/deployer_helm_tiller_base/test/post-install.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: v1
+kind: ServiceAccount
```

Comment:
I'm concerned about the creation of these service accounts and roles/role bindings. We're building our RBAC hacks into the deployer.

Since this should run while the deployer is still alive, this should use the deployer's Service Account.

We should introduce (yet!) a new property that takes in the deployer's service account and use it here.

Reply:
Agreed that we should avoid referencing cluster-admin, but I think we can downscope permissions further than the deployer's service account. We only need to fetch applications and patch in component kinds.

Let's discuss this and the post-delete case offline.

Reply:
Recapping offline discussion - a major reason for using the deployer service account here is to be consistent with disallowing partners from creating RBAC resources.

I have the impression that we disallow Partners creating RBAC resources that have privileges outside the application's namespace, but RBAC resources within the namespace are allowed.